pulsar-websocket/src/main/java/com/yahoo/pulsar/websocket/WebSocketService.java

```diff
@@ -177,6 +177,8 @@ private PulsarClient createClientInstance(ClusterData clusterData) throws IOExce
         clientConf.setTlsAllowInsecureConnection(config.isTlsAllowInsecureConnection());
         clientConf.setTlsTrustCertsFilePath(config.getTlsTrustCertsFilePath());
         clientConf.setIoThreads(WebSocketProxyConfiguration.PULSAR_CLIENT_IO_THREADS);
+        clientConf.setConnectionsPerBroker(WebSocketProxyConfiguration.PULSAR_CLIENT_IO_THREADS);
```

Comment:
Something seems off with the indentation

Can we create a separate parameter WebSocketProxyConfiguration.CONNECTIONS_PER_BROKER

Since user may want #IO threads as 10 but only one connection per broker.